docs/developers/linea-version/release-notes.md

```diff
@@ -0,0 +1,2283 @@
+---
+title: Release notes
+sidebar_position: 1
+---
+
+# Release notes
+
+Find out all you need to know for the latest Linea versions.
+
+## Linea Alpha v0.2.0 (June 6, 2023)
+
+### Summary
+
+This release focuses on testing a substantial architecture upgrade in preparation for Mainnet launch. It contains multiple improvements and breaking changes, specifically around the messaging layer which is changed to a push model. It also improves EVM prover coverage, and provides batch conflation.
```

Comment:
To be clear, are we hereby announcing that Mainnet is happening? Is that public news, yet?